cpp/include/cuml/tree/decisiontree.hpp

```diff
@@ -107,7 +113,8 @@ struct DecisionTreeParams {
 void set_tree_params(DecisionTreeParams &params, int cfg_max_depth = -1,
                      int cfg_max_leaves = -1, float cfg_max_features = 1.0f,
                      int cfg_n_bins = 8, int cfg_split_algo = SPLIT_ALGO::HIST,
-                     int cfg_min_rows_per_node = 2,
+                     int cfg_min_samples_leaf = 2,
```

Comment:
I think min_samples_leaf defaults to 1 in sklearn? https://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeClassifier.html